src/references/media_library.md

```diff
@@ -0,0 +1,88 @@
+---
+id: media_library_reference
+name: Media Library
+title: Media Library
+tags:
+  - Reference
+---
+
+# Media Library
+
+## Overview
+
+The **Media Library** in Core allows creators to upload content to their account which can be displayed in any of the creator's games.
+
+Content that can be uploaded such as images support JPEG, JPG, and PNG, along with support for transparency.
+
+Creators have a maximum total storage of 104 MB per account.
+
+### Opening the Media Library
+
+The Media Library can be opened by selecting **Media Library** from the **Window** menu. From this window creators can view, rename, and delete content they have uploaded to their account.
+
+Opening the Media Library window in different projects will show that creator's content.
```

Comment:
So the Media Library is shared across accounts and not stored on the project itself? Might need to reword this sentence to be more clear.